src/Models/AdaMultiLasso.cpp

```diff
@@ -45,7 +45,8 @@ void AdaMultiLasso::setAttributeMatrix(const string& str, MatrixXd* Z) {
     } else if (str == "snpsFeatures" || str == "snpsFeature") {
         setSnpsFeatures(*Z);
     } else {
-        throw runtime_error("AdaMultiLasso models have no attribute with name" + str);
+//        throw runtime_error("AdaMultiLasso models have no attribute with name" + str);
+        cerr << "AdaMultiLasso models have no attribute with name " << endl;
```

Comment:
because the frontend will deliver all the available extra files to it, if this project contains both extra files for adalasso, and multipop lasso, then both adalasso class and multipop lasso class will receive files for both of them. and this causes the running time error.